force-app/main/default/classes/ServiceScheduleService_TEST.cls

```diff
@@ -417,10 +418,7 @@ public with sharing class ServiceScheduleService_TEST {
                 new ServiceSession__c(
                     Name = sessionDateTime.date().format() + ': ' + schedule.Name,
                     SessionStart__c = sessionDateTime,
-                    SessionEnd__c = DateTime.newInstance(
-                        sessionDateTime.date(),
-                        firstSessionEnd.time()
-                    )
+                    SessionEnd__c = sessionDateTime.addHours(hours)
```

Comment:
This looks good, just curious why we needed to make this change?

Reply:
when run after midnight UTC, it would fail because of the way we were constructing the expected end datetime -- the old way not the new way with duration.